src/Tools/LeanCode.CodeAnalysis/Analyzers/CancellationTokensShouldFollowNamingConvention.cs

```diff
@@ -0,0 +1,66 @@
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace LeanCode.CodeAnalysis.Analyzers;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+public class CancellationTokensShouldFollowNamingConvention : DiagnosticAnalyzer
+{
+    internal const string IdentifierName = "CancellationToken";
+    internal const string ExpectedName = "cancellationToken";
```

Comment:
```suggestion
    internal const string TypeName = "CancellationToken";
    internal const string ParameterName = "cancellationToken";
```
What do you think? Are the names you used relate to CodeAnalysis API?